contents/handbook/people/team-structure/product-analytics/objectives.md

```diff
@@ -0,0 +1,91 @@
+- The further we continue down this path, the harder it'll be to have a fully integrated product across all data.
+
+Meta reason:
+- We now have separate product teams. 
+- If we're not careful, we'll end up with inoperable products. 
+- This builds the first shared bridge between all of them.
+
+### How?
+- Universal Search: https://github.com/PostHog/posthog/issues/7963
+- An "explore" interface/table, which supports input from various swappable data sources, with a unified set of filters applied to data.
+- Support for custom columns and [formulas](https://github.com/PostHog/posthog/issues/11913) on those tables.
+- Ability to save as a cohorts, save as an action, or export as a CSV when the data permits.
+- The ability to link to these tables from anywhere. "Explore further" links wherever we show *any* data in the interface.
+
+### Future steps.
+- Replace all other data tables in the app (events list, persons list, etc) with this interface.
+- Interconnect all the ideas here: https://github.com/PostHog/posthog/issues/3765
+- Add this power to the toolbar. E.g. add an action, see user recordings and explore further.
+- If we add a bit of text parsing as a follow-up project, this would lead the way to an interface like this:
+
+![image](https://user-images.githubusercontent.com/53387/191732592-569aa93e-7757-4f09-a109-3f7eb5eff2b7.png)
+
+## Proposed Objective 2: Nail Web Analytics
+- Why? A lot of our customers are only using us to track their product, and not their marketing website.
+  Our hypothesis is that marketing websites have 10-100x more events than applications.
+
+- Analysis:
+  - We are not seen as a web analytics company, and have previously worked to avoid that title.
+  - There is a large overlap between web and product analytics. Our infrastructure can handle both usecases.
+  - Google is sunsetting the traditional "google analytics" web analytics product ("universal analytics"), and moving
+    everyone to GA4. A lot of people are not happy with this, and are jumping ship. We could capture a lot of new users.
+  - If we build an analytics product that serves the needs of web marketers and integrates with the rest of our stack,
+    we're well suited for a lot of new growth.
+
+- Steps:
+  - Figure out why are 80% of our B2B customers not using us for their marketing website?
+    Is it missing features? Cost? Perception? Something else?
+  - What are the needs of typical marketing websites and marketers? A guess: better ad and source tracking.
+    Which of these are reasonable to build on our platform?
+  - Do we want to move into this segment, or should we avoid it? Is it a distraction from our mission of helping people
+    build better products? Might this result in a lot of low volume low quality free users?
+  - Actually implement what comes out of this analysis
+
+This objective includes a lot of product work to figure out what needs to be done. Thus it's the second objective
```

Comment:
I think there's a considerable risk that this appeals to a different persona (marketing rather than engineering) too, which could make it a lot harder to get adoption.

The alternative web analytic products are fairly commoditised (ie GA is _free_ and super entrenched), and there are multiple low cost bootstrappy alternatives. We'd need to think really hard why people would pick our web analytics.

Reply:
@jamesefhawkins I think the logic of the original goal of getting 80% of b2b website to add their marketing experiment to Posthog was: 
1. marketing websites have 10x-100x more event volume
2. therefore we should try and get the marketing websites also connected to Posthog.

When discussing with Marius, we noted that flows to increase the install rate is one part of the solution but likely the other part is we don't yet have a great web analytics product

However I agree the the persona does then likely shift from engineering to marketing as you noted.

To check I understand would you be pro a shorter push for activating b2b marketing websites but not deeper product work on the web analytics product? It's hard for me to see a version of getting great adoption of posthog on the marketing websites without putting in more time into the marketing persona. But I could imagine getting good adoption fairly quickly

Reply:
It's possible we can get 80% of b2b customers to add their marketing website and send us 10x more events/money by asking "why not?", but I suspect we need a better web analytics offering to actually service this market. Thus we need to figure out what we don't know, what we need to build to win users, and how to best competitors in this field. That's a lot of research that needs to take up someone's time.

However, I would personally scrap this objective.

From my perspective, this came from nowhere, hasn't been directly requested by any user I know of, and feels like something you'd build if you wanted to show great growth numbers before a fundraise... not if you listened to users feedback on what product they want from us in order to build great products themselves 🤷. I would personally tackle issues closer to "team product analytics" before effectively working on a new product 😅.

I included it as an OK as it was effectively communicated as request from the execs team, and I'm willing to work on it if needed.